pkg/agent/agent_windows.go

```diff
@@ -176,6 +176,15 @@ func (i *Initializer) prepareOVSBridge() error {
 		klog.Errorf("Failed to set the uplink port with no-flood config: %v", err)
 		return err
 	}
+
+	// If noencap/hybrid mode, br-int needs enabling IP forwarding for return traffic.
+	if i.networkConfig.TrafficEncapMode == config.TrafficEncapModeNoEncap ||
+		i.networkConfig.TrafficEncapMode == config.TrafficEncapModeHybrid {
+		if err := util.EnableIPForwarding(brName); err != nil {
+			klog.Errorf("Failed to enable IP forwarding on interface %s: %v", brName, err)
```

Comment:
I meant do we need to fail the agent startup here? If so, we should return the error (not just log).

Reply:
Thanks. Code updated.
Actually your latest comment referred to another place using `klog` but yes both places need update . https://github.com/vmware-tanzu/antrea/pull/1901#discussion_r616875856